packages/types/src/transaction.ts

```diff
@@ -169,3 +175,17 @@ export type TransactionSource =
   | 'component'
   /** Name of a background task (e.g. a Celery task) */
   | 'task';
+
+/**
+ * Object representing metadata about when a transaction name was changed.
+ */
+export interface TransactionNameChange {
+  // new name
+  name: string;
```

Comment:
As discussed offline, we don't require the name for analysis right now. If we drop it, we further prevent the possibility of capturing unintended.

Reply:
done - https://github.com/getsentry/sentry-javascript/pull/5709/commits/3ef0b80739218140e45e8ecfb13e63412800227d